day5/pathsum.py

```diff
@@ -0,0 +1,24 @@
+# Definition for a binary tree node.
+# class TreeNode(object):
+#     def __init__(self, x):
+#         self.val = x
+#         self.left = None
+#         self.right = None
+
+class Solution(object):
+    def hasPathSum(self, root, sum):
```

Comment:
I thought as much but this was the skeleton code in the leetcode page. But yeah you're right.